src/Umbraco.Web.UI.Client/src/common/services/formhelper.service.js

```diff
@@ -57,6 +61,30 @@ function formHelper(angularHelper, serverValidationManager, notificationsService
             return true;
         },
 
+         /**
+         * @ngdoc function
+         * @name umbraco.services.formHelper#focusOnFirstError
+         * @methodOf umbraco.services.formHelper
+         * @function
+         *
+         * @description
+         * Called by submitForm when a form has been submitted, it will fire a focus on the first found invalid umb-property it finds in the form..
+         * 
+         * @param {object} form Pass in a form object.
+         */
+        focusOnFirstError: function(form) {
+            var invalidNgForms = form.$$element.find(`.umb-property ng-form.ng-invalid, .umb-property-editor ng-form.ng-invalid-required`);
```

Comment:
Heyo, as mentioned below, the quotes are template literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), native JS. I'm happy to change them to standard quotes though.

In regards to the `$$element`, it is a private property on an angular element, so in hindsight I do think it should be changed. I may need to spend some more time working on a better solution. I was using `$$element` in order to access the form angular element because I wasn't able to do a `find()` or `querySelector` on the `form` value that it provided by the form submit.

Reply:
I learned something new.. template literals, got it! I think if it works it might be fine for now (surprise: it works 😉).